File: exercises/beecrowd/1050/1050.go
// https://judge.beecrowd.com/pt/problems/view/1050
package main

import (
	"fmt"
)

func main() {
	var ddd int

	fmt.Scanln(&ddd)

	dddMap := map[int]string{
		61: "Brasilia",
		71: "Salvador",
		11: "Sao Paulo",
		21: "Rio de Janeiro",
		32: "Juiz de Fora",
		19: "Campinas",
		27: "Vitoria",
		31: "Belo Horizonte",
	}

	if city, ok := dddMap[ddd]; ok {
		fmt.Println(city)
	} else {
		fmt.Println("DDD nao cadastrado")
	}
}

File: exercises/beecrowd/1050/1050.java
// https://judge.beecrowd.com/pt/problems/view/1050
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int ddd = sc.nextInt();

        String city;
        switch (ddd) {
            case 61: city = "Brasilia"; break;
            case 71: city = "Salvador"; break;
            case 11: city = "Sao Paulo"; break;
            case 21: city = "Rio de Janeiro"; break;
            case 32: city = "Juiz de Fora"; break;
            case 19: city = "Campinas"; break;
            case 27: city = "Vitoria"; break;
            case 31: city = "Belo Horizonte"; break;
            default: city = "DDD nao cadastrado"; break;
        }

        System.out.println(city);
        sc.close();
    }
}
